WcaOnRails/app/webpacker/components/CompetitionForm/FormInputs.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable react/no-danger */
+import React, { useState, useCallback } from 'react';
+
+import {
+  Checkbox,
+  Form,
+  Input,
+} from 'semantic-ui-react';
+import I18n from '../../lib/i18n';
+
+// Modified from '../../lib/hooks/useInputState';
```

Comment:
I haven't checked the original hook, but what's the difference? Any chance the other hook could be refactored instead of making a new one? If not, should it be moved to the hooks folder too?